src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: Provide a custom install experience
+authors:
+  - petelepage
+date: 2020-02-14
+updated: 2020-02-14
+description: |
+  Use the beforeinstallprompt event to provide a custom, seamless install
+  experience for your users.
+tags:
+  - post
+draft: true
+---
+
+Many browsers make it possible for you to promote and enable the installation
+of your Progressive Web App (PWA) directly within the user interface of your
+PWA. Installation (sometimes formerly referred to as Add to Home Screen),
+makes it easy for users to install your PWA on their mobile or desktop device.
+Installing a PWA adds it to a user's launcher, allowing it to be run like any
+other installed app.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="spotify-custom-install.png"
+       alt="Install App button provided in the Spotify PWA">
+  <figcaption class="w-figcaption">
+    "Install App" button provided in the Spotify PWA
+  </figcaption>
+</figure>
+
+Why would you want a user to install your app? The same reason you'd want a
+user to install your app from any app store. Users who install are your most
+engaged audience. Users who install a PWA have better engagement metrics than
```

Comment:
I think 'audience' actually needs to be replaced with 'users'. Good writers try to avoid repeating a word in proximity to itself; however there are times when it's appropriate. I think this is one of them.

There's also a technical issue that hinges on the distinction between a plural form and a collective noun. A collective noun has both a singular and a plural form. ("The **audience** enjoyed the movie." vs. "The multiplex's air conditioning was out so the **audiences** were uncomfortable.") 'Audience' is a collective noun, but is grammatically singular. You've equated it with a plural noun.

Also, 'audience' suggests content distribution use cases while PWA use cases are broader.

Reply:
Fixed!